tools/modules/software/module_homepage.sh

```
@@ -0,0 +1,84 @@
+	["module_homepage,port"]="3000"	
+	["module_homepage,arch"]=""
+)
+#
+# Module homepage
+#
+function module_homepage () {
+	local title="homepage"
+	local condition=$(which "$title" 2>/dev/null)
+
+	if pkg_installed docker-ce; then
+		local container=$(docker container ls -a | mawk '/homepage?( |$)/{print $1}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)